### DISCUSSION TITLE
Audio/Sound during Sharescreen and ideas.

### DISCUSSION THREAD

**Discussion by jtxxviii** [netless-io/flat#1845] [Ideas]
Hi Agora Flat team,

I thought it would be nice if we could hear sound/audio during "share screen".
我想如果我们能在“共享屏幕”期间听到声音/音频就好了。

And I want to share my ideas.
1. Camera placement  相机放置
A. Side
![1](https://user-images.githubusercontent.com/126577964/221871398-5fba5a60-1576-4cf6-8b0e-6b9711884d41.PNG) （请检查）Please check 
 or
B. Top 
![2](https://user-images.githubusercontent.com/126577964/221873129-3d26b5c6-fca1-4217-a47e-19c4d68baadd.PNG) （请检查）Please check 
 
3. Can add or remove contact/user     可以添加或删除联系人/用户
4. Can create specific room/group 可以创建特定的房间/组
      a. able to invite contact user ( add/ remove ) 能够邀请联系人用户
      b. can create scheduled lessons/meetings 能够创建预定的课程/课程
     ![C](https://user-images.githubusercontent.com/126577964/221876260-df5e6726-d1d0-4db0-8c6d-0fbfda5a6a8f.PNG) （请检查）Please check 
  
Thanks!

**Comment by Leooeloel** [comment]
Thanks for the feedback, this is a good suggestion, we will carry out the implementation, but it will take some time.
